docs/eval/s5-claim-evidence.md

```
@@ -0,0 +1,220 @@
+| **S5-D13** | Order: eligibility → pack missing → judge | runner disposition order tests |
+| **S5-D14** | Retry classes bounded/classified | judge retry-once transport/parse; never empty |
+| **S5-D15** | `slice0_baseline.json` machine baseline | `docs/eval/slice0_baseline.json` |
+| **S5-D16** | Gate↔execution taxonomy mapping | `TestTaxonomy::*` |
+| **S5-D17** | `score_bundle` isolates C′ exceptions | scoring runner try/except + immunity tests |
+
+### S5-E — runner wiring + docs spine
+
+| ID | Summary | Primary evidence |
+|:---|:---|:---|
+| **S5-E01** | Judge only on eligible+available path | runner invoke vs skip matrix in `test_lane_c.py` |
+| **S5-E02** | Craft/relevance packs present | `prompts/eval/lane_c/geval_craft|relevance/rubric.md` + pack tests |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)